homeassistant/components/apple_tv/strings.json

```
@@ -62,6 +62,17 @@
       "inconsistent_device": "Expected protocols were not found during discovery. This normally indicates a problem with multicast DNS (Zeroconf). Please try adding the device again."
     }
   },
+  "entity": {
+    "binary_sensor": {
+      "keyboard_focused": {
+        "name": "Keyboard focused",
+        "state": {
+          "on": "Focused",
+          "off": "Not focused"
```

For "off" this results in "**Keyboard focused: Not focused**" which is making this quite complicated, even more in different translations.

Rather just use "**Keyboard focus: Off**" instead.

```suggestion
        "name": "Keyboard focus",
        "state": {
          "on": "[%key:common::state::on%]",
          "off": "[%key:common::state::off%]"
```

---

Waiting on the answer below.